src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs

```diff
@@ -73,11 +73,12 @@ protected async ValueTask<T> RunServiceAsync<T>(Func<CancellationToken, ValueTas
             WorkspaceManager.SolutionAssetCache.UpdateLastActivityTime();
             using var _ = LinkToken(ref cancellationToken);
 
+            // TODO: check cancellationToken in the filter https://github.com/dotnet/roslyn/issues/48175
             try
             {
                 return await implementation(cancellationToken).ConfigureAwait(false);
             }
-            catch (Exception ex) when (FatalError.ReportWithoutCrashUnlessCanceledAndPropagate(ex, cancellationToken))
+            catch (Exception ex) when (FatalError.ReportWithoutCrashUnlessCanceledAndPropagate(ex))
```

Comment:
📝 The `CancellationToken` here is only used to determine whether or not a non-fatal error is reported through telemetry. It does not impact the return value from this method or the runtime code flow. The old code would do a better job of alerting us to cases where cancellation is reported unexpectedly, so it seems like we would not want to change this.

Reply:
You are right, reverting this one.